javascript/updates_for_element.js

```diff
@@ -13,12 +13,16 @@ const template = `
 <slot></slot>
 `
 
+function url (ele) {
+  return ele.hasAttribute('url') ? ele.getAttribute('url') : location.href
+}
+
 class UpdatesForElement extends HTMLElement {
   constructor () {
     super()
     const shadowRoot = this.attachShadow({ mode: 'open' })
     shadowRoot.innerHTML = template
-    this.update = Boris(this.update.bind(this), this.debounce)
+    this.update = debounce(this.update.bind(this), this.debounce)
```

Comment:
I just noticed a regression here, but this has probably already crept up during the last refactoring:

There's no chance the element can have a `debounce` attribute at the time of construction, i.e. `this.debounce` will always return 20. So in my case I'm seeing thundering herds again.

I think we need to move this to `connectedCallback`

Reply:
I think that you're confusing the `debounce` function that is being imported with the `this.debounce` getter available inside the component.